src/components/pages/Search/RenderSearchPage.js

```diff
@@ -28,16 +52,29 @@ const Searching = () => {
   }, []);
 
   const handleChange = event => {
+    console.log('getting the event:', event);
     setSearchValue(event.target.value);
   };
 
+  // const milesChange = value => {
```

Comment:
make sure if you leave this code in here, that you leave a comment next to all the commented out code stating what they are used for. If they do not need to be there, then delete all the commented out code.

Reply:
Hi Sara , i deleted all commented out code and submited new PR.